packages/bot-web-ui/src/stores/chart-store.js

```diff
@@ -19,7 +19,6 @@ export default class ChartStore {
 
     @observable symbol;
     @observable is_chart_loading;
-    @observable main_barrier = null;
```

Comment:
Is this intentional?

Reply:
Yes, we don't use and update this. In observable it's only set as null